be/src/storage/tablet_schema.cpp

```diff
@@ -781,6 +781,15 @@ bool operator!=(const TabletSchema& a, const TabletSchema& b) {
     return !(a == b);
 }
 
+bool TabletSchema::has_separate_sort_key() const {
+    for (size_t i = 0; i < _sort_key_idxes.size(); ++i) {
+        if (_sort_key_idxes[i] != i) {
+            return true;
+        }
+    }
+    return false;
```

Comment:
Should we check the `sort_key_column` size is equal to `num_key_columns`?

Reply:
done